ceph/ceph_admin/typing_.py

```diff
@@ -75,6 +75,15 @@ def op(self, op: str, config: Dict):
     def verify_status(self, op: str) -> None:
         ...
 
+    def get_host(self, hostname: str) -> bool:
+        ...
+
+    def check_maintenance_status(self, op, node: CephNode) -> bool:
+        ...
+
+    def get_host_status(self, hostname: str) -> str:
+        ...
+
```

Comment:
As these methods are available as part of mixins, we don't need to specify them in the Protocol class

Reply:
Done.